CONTRIBUTING.md

```diff
@@ -1,7 +1,24 @@
 # Contributing
 
-You can take part in making Shopsys Framework better.
+You can take part in making Shopsys Framework Demoshop better.
 
-Report [issues](https://github.com/shopsys/shopsys/issues/new) and send [pull requests](https://github.com/shopsys/shopsys/compare) in the main [Shopsys repository](https://github.com/shopsys/shopsys).
+* [Create a pull request](https://github.com/shopsys/demoshop/compare)
+* [Report an issue](https://github.com/shopsys/demoshop/issues/new)
 
-Please, check our [Contribution Guide](https://github.com/shopsys/shopsys/blob/master/CONTRIBUTING.md) before contributing.
+To make your contribution easier for us to process, please follow guidelines for contributing in our main repository [shopsys/shopsys](https://github.com/shopsys/shopsys):
+* [Guidelines for Creating Commits](https://github.com/shopsys/shopsys/blob/master/docs/contributing/coding-standards.mddocs/contributing/guidelines-for-creating-commits.md)
+* [Guidelines for Writing Documentation](https://github.com/shopsys/shopsys/blob/master/docs/contributing/coding-standards.mddocs/contributing/guidelines-for-writing-documentation.md)
+* [Guidelines for Pull Request](https://github.com/shopsys/shopsys/blob/master/docs/contributing/coding-standards.mdocs/contributing/guidelines-for-pull-request.md)
+* [Guidelines for Dependencies](https://github.com/shopsys/shopsys/blob/master/docs/contributing/coding-standards.mddocs/contributing/guidelines-for-dependencies.md)
+
+*Note: Guidelines can contain references and rules applicable only for main repository shopsys/shopsys. In that case, please follow the sprit of these rules and guidelines.*
+
+For your code to be accepted, you should follow our guidelines mentioned above,
```

Comment:
I wouldnt mention guidelines again. "For accepting your contribution, your code must pass the coding standards checks and tests [link]"

Reply:
This sentence is in our monorepo contribution.md and I want to keep it consistent. So I prefer to keep it this way.